plugins/modules/postgresql_user.py

```diff
@@ -21,6 +21,7 @@
 - Set I(fail_on_user) to C(no) to make the module ignore failures when trying to remove a user.
   In this case, the module reports if changes happened as usual and separately reports
   whether the user has been removed or not.
+- B(WARNING) The C(priv) option has been B(deprecated) and will be removed in community.postgresql 3.0.0. Please use the I(postgresql_privs) module instead.
```

Comment:
```suggestion
- B(WARNING) The I(priv) option has been B(deprecated) and will be removed in community.postgresql 3.0.0. Please use the M(community.postgresql.postgresql_privs) module instead.
```
this is a requirement from the official doc, i.e. modules have to use M(FQCN)